chapters/operatorsandexpressions.tex

```diff
@@ -440,7 +442,8 @@ \subsection{Numeric Functions and Conversion Operators}\label{numeric-functions-
 \end{tabular}
 \end{center}
 
-All of these except for the \lstinline!String! conversion operator are vectorizable according to \cref{scalar-functions-applied-to-array-arguments}.
+Except for the \lstinline!min!, \lstinline!max! and the \lstinline!String! conversion operator they are vectorizable according to \cref{scalar-functions-applied-to-array-arguments}.
+The \lstinline!min! and \lstinline!max! functions including additional variants are described in \cref{reduction-functions-and-operators}.
```

Comment:
I find _including additional variants_ a bit unclear, could it be reformulated?

Reply:
Suggestion: The \lstinline!min! and \lstinline!max! functions have array-specific variants which perform array reduction operations described in ...